x-pack/platform/plugins/shared/onechat/public/application/hooks/use_conversation_list.ts

```diff
@@ -18,7 +18,7 @@ export const useConversationList = ({ agentId }: { agentId?: string }) => {
     refetch: refresh,
   } = useQuery({
     queryKey: agentId ? queryKeys.conversations.byAgent(agentId) : queryKeys.conversations.all,
-    queryFn: async () => {
+    queryFn: () => {
```

Comment:
is there a way we could try prefetch the conversation list earlier vs on the render? Theres a slight delay which isn't great UX 

https://github.com/user-attachments/assets/e52c1d7d-cd4f-4bb5-aa7e-5038b6c1b9c8